run_dir/design/flowcells.html

```diff
@@ -262,13 +262,26 @@ <h1><span id="illumina_page_title">Illumina flowcells</span>
       "paging":false,
       "info":false,
       "order": [[ 1, "desc" ]],
+      colReorder: true,
       dom: 'Bfrti',
       buttons: [
         { extend: 'copy', className: 'btn btn-outline-dark mb-3' },
         { extend: 'excel', className: 'btn btn-outline-dark mb-3' }
       ]
     });
 
+    //Keep markdown format when dragging around the columns
+    table.on( 'column-reorder', function() {
+        $(".running-note-card > .card-body").each(function(i){
+            $(this).html(make_markdown($(this).text()));
+        });
+        $('.fillbadgecolour').html(function(){
+            let categories = JSON.parse($(this).text().replace(/'/g, '"'))
+            //from running_notes.js
+            return generate_category_label(categories)
+        })
```

Comment:
Put these lines in a function, maybe called `format_running_note` and call it here and line 244? The code is repeated now.

Reply:
Good idea!

Reply:
On stage again :-)